docs/migration/v4-migration.md

```diff
@@ -37,6 +37,13 @@ A number of changes were made to the configuration options passed to the `Chart`
 * The order of the `ChartMeta` parameters have been changed from `<Element, DatasetElement, Type>` to `<Type, Element, DatasetElement>`.
 
 ### General
+* Chart.js becomes an [ESM-only package](https://nodejs.org/api/esm.html). To use Chart.js, your project should also be an ES module. Make sure to have this in your `package.json`:
```

Comment:
This is not totally true, UMD build is still available to use. Either by using script tag or as you can see in the solution to this issue by targeting the UMD build.

https://github.com/chartjs/Chart.js/issues/10875

So maby word is something like chartjs has become esm package by default

Reply:
done